platform/source/minimal-printf/README.md

```diff
@@ -12,10 +12,10 @@ Supports:
 * %u: unsigned integer [h, hh, (none), l, ll, z, j, t].
```

Comment:
We should addnote that ll -variants are ONLY supported if `minimal-printf-enable-64-bit`  is enabled. We've already had our run with this issue in the client side. LwM2M specification states that int == int64_t... 

Also, in that case it failed and printed just `lld`, would be nice and more convenient to spot the issue if the % sign would have also been left there...